src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * The TaskList contains the list of tasks and operations to modify the list.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    /**
+     * Constructor for a TaskList object with an existing list.
+     * @param taskList An arraylist of Tasks.
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Constructor for a TaskList object with a new list.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    /**
+     * Gets the list in the TaskList object.
+     * @return An arraylist containing all the Tasks.
```

Comment:
Perhaps you should leave a line between your description and your @return line?

Reply:
Alright thanks